pkg/config/config_template.yaml

```diff
@@ -105,6 +105,13 @@ api_key:
 # IPC api server timeout in seconds
 # server_timeout: 15
 
+# In some environments we may have the procfs file system mounted in a
```

Comment:
nit: I think we want to avoid `we` in docs ;) So maybe `In some environments the procfs file system may be mounted [...]` instead?